front/src/components/atoms/IconButton/IconButton.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles, createStyles, Theme } from "@material-ui/core/styles";
+import IconButton from "./IconButton";
+
+export default {
+  title: "Atom/IconButton",
+  component: IconButton,
+};
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      color: theme.palette.text.primary,
+    },
+  })
+);
+
+export const IconButton = () => {
```

Comment:
```suggestion
export const BasicIconButton = () => {
```
컴포넌트 IconButton과 이름이 같아 스토리북에 조회되지 않습니다.

Reply:
수정했습니다!